src/Module/MailCatcher.php

```diff
@@ -365,7 +366,7 @@ protected function emailFromId($id)
      */
     protected function seeInEmailSubject(Email $email, $expected)
     {
-        $this->assertContains($expected, $email->getSubject(), "Email Subject Contains");
+        $this->assertStringContainsString($expected, $email->getSubject(), "Email Subject Contains");
```

Comment:
This change breaks backwards compatibility with Codeception 2. `assertStringContainsString` was added in v3.

Either update `composer.json` to allow 2.1 of this package to only require v3 or update this method to be backwards compatible.